app/models/book.rb

```diff
@@ -0,0 +1,5 @@
+class Book < ApplicationRecord
+	validates_presence_of :title, :category
```

Comment:
Why don't we try using 

```ruby
validates :title, :category, :read, presence: true
```

It's not adding `read` to the validations that won't make it work. It's the fact that we cannot do any successful requests anywhere because we aren't passing in any authenticated user to the request.